### PR TITLE
Support for go struct tags

### PR DESCRIPTION
**Pull request by jeroenrinzema**
This PR includes the support for the definition/generation of struct tags inside AVRO schemas as discussed in #76.
Tags are defined in a `go_tags` attribute and are appended to the end of a struct field.

**Example:**
```json
{
  "name" : "StructTag",
  "type" : "record",
  "fields" : [
    {
      "name" : "productName",
      "type" : "string",
      "go_tags": "validate:\"true\""
    }
  ]
}
```

**Generates:**
```golang
type StructTag struct {
  ProductName string `validate:"true"`
}
```

### CODE REVIEW

**Comment by jeroenrinzema** [comment]
I just read your blog post about [GADGT](http://www.agardner.me/golang/avro/vm/gadgt/go/deserializer/2019/02/15/gdgt-future-of-gogen-avro.html) if you like could I also create a PR to the `gadgt` branch

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module github.com/actgardner/gogen-avro
```
Comment (actgardner) [inline go.mod]: Can you exclude these from the PR? I haven't really caught up on the module system, I'd rather add that separately.

Comment (jeroenrinzema) [reply go.mod]: I have removed the go.mod and go.sum files

**Comment by jeroenrinzema** [comment]
@actgardner I have updated the PR but let's wait till the outcome of #77 before merging.

**Comment by jeroenrinzema** [comment]
@actgardner I have renamed the tags property to `go.tags` what do you think?

**Comment by actgardner** [comment]
I like it! Thanks for your work on this, can you update the tests so they pass CI and I'll merge it?

**Comment by jeroenrinzema** [comment]
I have already created a new test (`go-struct-tags`) but forgot to update the struct tag inside the tests. Feel free to create additional tests.

**Comment by maarek** [comment]
This looks good to me! 👍

**Pull request merged**